<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="ctrl_wid" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>700</y>
      <width>574</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,30,1,5">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="play_LB">
       <property name="text">
        <string>00:00:00 / 00:00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="play_BTN">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>||</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="play_SLD">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>  Vol:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>26</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="XVideoWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>769</width>
      <height>539</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>rtmp://58.200.131.2:1935/livetv/gdtv</string>
    </property>
   </widget>
   <zorder>openGLWidget</zorder>
   <zorder>ctrl_wid</zorder>
   <zorder>lineEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuopenFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionopen_file"/>
    <addaction name="actionopen_url"/>
   </widget>
   <addaction name="menuopenFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen_file">
   <property name="text">
    <string>open file</string>
   </property>
  </action>
  <action name="actionopen_url">
   <property name="text">
    <string>open_url</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XVideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">XVideoWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>play_SLD</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>SliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>747</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>871</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>play_SLD</sender>
   <signal>sliderPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>SliderPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>747</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>878</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>SetVolume(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>768</x>
     <y>743</y>
    </hint>
    <hint type="destinationlabel">
     <x>806</x>
     <y>895</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SliderRelease()</slot>
  <slot>SliderPressed()</slot>
  <slot>SetVolume(int)</slot>
 </slots>
</ui>
